setup.py

```diff
@@ -20,6 +20,7 @@ def get_version() -> str:
     "requests",
     "tqdm>=4.42.1",
     "typing-extensions>=3.7.4.3",  # to be able to import TypeAlias
+    "dotenv",
```

Comment:
I hope this is fine as a new dependency - I'd have loved to not rely on anything but env variables parsing seems non trivial unfortunately

Reply:
There are no dependencies for the package (unless you install with a [cli] extras flag).  Source: https://libraries.io/pypi/python-dotenv + https://github.com/theskumar/python-dotenv/blob/16f2bdad2ebbaae72790514cce713d2d22ab0f7c/setup.py#L41